games-simulation/openrct2/openrct2-9999.ebuild

```diff
@@ -1,113 +1,164 @@
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit cmake-utils gnome2-utils xdg-utils
+inherit cmake-utils git-r3 readme.gentoo-r1 xdg-utils
 
-DESCRIPTION="An open source re-implementation of RollerCoaster Tycoon 2"
+EGIT_REPO_URI="https://github.com/OpenRCT2/OpenRCT2.git"
+EGIT_BRANCH="develop"
+
+MY_PN="OpenRCT2"
+MY_PN_OBJ="objects"
+MY_PN_RPL="replays"
+MY_PN_TS="title-sequences"
+MY_PV_OBJ="1.0.12"
+MY_PV_RPL="0.0.4"
+MY_PV_TS="0.1.2c"
+
+DESCRIPTION="An open source re-implementation of Chris Sawyer's RollerCoaster Tycoon 2"
 HOMEPAGE="https://openrct2.org/"
-if [[ ${PV} == 9999 ]]; then
-	EGIT_REPO_URI="https://github.com/OpenRCT2/OpenRCT2.git"
-	EGIT_BRANCH="develop"
-	inherit git-r3
-	SRC_URI=""
-else
-	KEYWORDS="~amd64 ~x86"
-	SRC_URI="https://github.com/OpenRCT2/OpenRCT2/archive/v${PV}.tar.gz
-		-> ${P}.tar.gz"
-	S="${WORKDIR}/OpenRCT2-${PV}"
-fi
-
-TSV="0.1.2"
-OBJV="1.0.8"
-SRC_URI+="
-	https://github.com/OpenRCT2/title-sequences/releases/download/v${TSV}/title-sequence-v${TSV}.zip
-		-> ${PN}-title-sequence-v${TSV}.zip
-	https://github.com/OpenRCT2/objects/releases/download/v${OBJV}/objects.zip
-		-> ${PN}-objects-v${OBJV}.zip"
+SRC_URI="
+	https://github.com/${MY_PN}/${MY_PN_OBJ}/releases/download/v${MY_PV_OBJ}/${MY_PN_OBJ}.zip -> ${PN}-${MY_PN_OBJ}-${MY_PV_OBJ}.zip
+	https://github.com/${MY_PN}/${MY_PN_TS}/releases/download/v${MY_PV_TS}/${MY_PN_TS}.zip -> ${PN}-${MY_PN_TS}-${MY_PV_TS}.zip
+	test? ( https://github.com/${MY_PN}/${MY_PN_RPL}/releases/download/v${MY_PV_RPL}/${MY_PN_RPL}.zip -> ${PN}-${MY_PN_RPL}-${MY_PV_RPL}.zip )
+"
 
 LICENSE="GPL-3"
 SLOT="0"
-IUSE="libressl +multiplayer opengl test truetype +twitch"
+KEYWORDS=""
+IUSE="dedicated libressl +lightfx +opengl test +truetype twitch"
 
-RDEPEND="
-	>=dev-libs/jansson-2.5
-	>=dev-libs/libzip-1.0
+COMMON_DEPEND="
+	dev-libs/icu:=
+	dev-libs/jansson
+	dev-libs/libzip:=
 	media-libs/libpng:0=
-	media-libs/libsdl2
-	media-libs/speexdsp
-	multiplayer? (
-		libressl? ( dev-libs/libressl:0= )
-		!libressl? ( dev-libs/openssl:0= )
+	sys-libs/zlib
+	!dedicated? (
+		media-libs/libsdl2
+		media-libs/speexdsp
+		opengl? ( virtual/opengl )
+		twitch? (
+			net-misc/curl[ssl]
+		)
 	)
-	opengl? ( virtual/opengl )
+	libressl? ( dev-libs/libressl:0= )
+	!libressl? ( dev-libs/openssl:0= )
 	truetype? (
-		media-libs/sdl2-ttf
-		media-libs/fontconfig
+		media-libs/fontconfig:1.0
+		media-libs/freetype:2
 	)
-	twitch? ( net-misc/curl[ssl] )
 "
-DEPEND="${RDEPEND}
-	app-arch/unzip
+
+RDEPEND="
+	${COMMON_DEPEND}
+	dedicated? (
+		acct-group/openrct2
+		acct-user/openrct2
+	)
+"
+
+DEPEND="
+	${COMMON_DEPEND}
 	test? ( dev-cpp/gtest )
 "
 
+BDEPEND="
+	app-arch/unzip
+	virtual/pkgconfig
+"
+
+RESTRICT="!test? ( test )"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-0.2.4-include-additional-paths.patch"
+)
+
 src_unpack() {
-	if [[ ${PV} == 9999 ]]; then
-		git-r3_src_unpack
-	else
-		unpack ${P}.tar.gz
-	fi
+	einfo $(cat /etc/resolv.conf)
```

Comment:
Is this some debug statement that you forgot to remove?

Reply:
Yeah. My ThinkPad died here, and my backup didn't worked in -9999. But I found the cause, sorry for that.